packages/renderer/src/lib/pod/PodsList.svelte

```
@@ -104,8 +104,7 @@ async function deleteSelectedPods(): Promise<void> {
 
   // mark pods for deletion
   bulkDeleteInProgress = true;
-  selectedPods.forEach(pod => (pod.status = 'DELETING'));
-  pods = pods;
+  selectedPods.forEach(pod => setPodStatus(pod.engineId, pod.id, 'DELETING'));
```

Biome (2.5.3)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)